.github/workflows/build_main.yml

```
@@ -1,83 +1,109 @@
+          ditto -c -k --keepParent "${BUNDLE}.bundle" "${BUNDLE}.zip"
+          "${{ env.XCODE_PATH }}/Contents/Developer/usr/bin/notarytool" submit \
+            "${BUNDLE}.zip" \
+            --keychain-profile "default" \
+            --wait
+          "${{ env.XCODE_PATH }}/Contents/Developer/usr/bin/stapler" staple \
+            "${BUNDLE}.bundle"
+          rm "${BUNDLE}.zip"
 
       - name: Set environment variables
         id: set_env_var
         run: |
```

actionlint

shellcheck reported issue in this script: SC2129:style:2:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)